src/app/@themes/styles/variable.scss

```diff
@@ -0,0 +1,11 @@
+$dark-100: #0B090A;
+$dark-200: #171A1D;
+$primary-100: #5D130E;
+$primary-200: #972722;
+$primary-300: #AB2C26;
+$primary-400: #D34843;
+$light-100: #737373;
+$light-200: #D3D3D3;
+$light-300: #F5F3F4;
+$light-400: #fff;
```

Comment:
if you'll import this file to another - it's better to rename file to _variables